python/searchSystemPackageSoftware.py

```diff
@@ -0,0 +1,33 @@
+# list software from the system package
+# ex: python3 searchSystemPackageSoftware.py http://192.168.13.111:8080 companyinfra openrmfprosvc hvs.xxxxxxxxxxxx
```

Comment:
Need to show passing in the software name argument like `Microsoft Office` or something.